ops/docker-sequencer-test/run-test.sh

```
@@ -0,0 +1,553 @@
+        --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' \
+        "$rpc_url" 2>/dev/null)
+    echo "$result" | grep -o '"result":"[^"]*"' | cut -d'"' -f4 | xargs printf "%d" 2>/dev/null || echo "0"
+}
+
+wait_for_block() {
+    local target_height=$1
+    local rpc_url="${2:-$L2_RPC}"
+    
+    log_info "Waiting for block $target_height..."
+    while true; do
+        local current=$(get_block_number "$rpc_url")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)